src/IntercomClient.php

```diff
@@ -265,6 +265,31 @@ public function nextPage($pages)
         return $this->handleResponse($response);
     }
 
+        /**
+     * Returns the next page of the result for a cursor based search.
+     *
+     * @param  stdClass $pages
+     * @return stdClass
+     */
+    public function nextSearchPage($path, $query, $pages)
+    {
+        $options = [
+            "query" => $query,
+            "pagination" => [
+                "per_page" => $pages->per_page,
+                "starting_after" => $pages->next->starting_after,
+            ]
+        ];
+        $response = $this->sendRequest('POST', "https://api.intercom.io/$path", $options);
```

Comment:
In order to avoid hardcoding the URL in multiple places, why don't we call `$this->post` here?

```suggestion
        $response = $this->post($path, $options);
```

Same for the next method